watson-developer-cloud/discovery-components
# fix: recreate section fields (highlights) on component resizing

#### What do these changes do/fix?

(New version of #92)

Recreate the section fields (highlights) whenever component resizes, to ensure that they line up correctly with the text.

https://github.ibm.com/Watson-Discovery/the-tooling-core-tracker/issues/1122

#### How do you test/verify these changes?

There's only one way that I've been able to reliably hit this bug, so this is probably the best way to test the fix as well:

1. `yarn link` `discovery-components` to your local `discovery-tooling`.
2. Run the Cypress tests for the enrichment view within `documentDetails.spec.js` (https://github.ibm.com/Watson-Discovery/discovery-tooling/pull/7108)
3. Note in the enrichment view screenshots that the enrichments are properly aligned with the highlighting blocks.

(You can also verify that without these changes, the enrichments are misaligned.)

#### Have you documented your changes (if necessary)?

#### Are there any breaking changes included in this pull request?
